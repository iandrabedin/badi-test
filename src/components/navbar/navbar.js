import React from "react";
import "./navbar.scss";

export const Navbar = () => {
  return (
    <div className="navbar">
      <div className="container">Recipe Puppy</div>
    </div>
  );
};
