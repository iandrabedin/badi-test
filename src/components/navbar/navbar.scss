@import "../../styles/variables.scss";

.navbar {
  display: flex;
  align-items: center;
  width: 100%;
  height: 64px;
  background-color: $color-primary;
  color: $color-white;
  font-size: $font-size-x-large;
  font-weight: $font-weight-semibold;
}
