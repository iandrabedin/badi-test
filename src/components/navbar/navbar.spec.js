import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Navbar } from "./navbar";

afterEach(cleanup);

describe("Navbar", () => {
  it("should render as expect", () => {
    const { asFragment } = render(<Navbar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
