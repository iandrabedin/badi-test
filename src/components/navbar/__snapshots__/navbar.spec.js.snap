// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render as expect 1`] = `
<DocumentFragment>
  <div
    class="navbar"
  >
    <div
      class="container"
    >
      Recipe Puppy
    </div>
  </div>
</DocumentFragment>
`;
